src/lib/transition_frontier/persistent_frontier/persistent_frontier.ml

```diff
@@ -151,7 +151,8 @@ module Instance = struct
       Error `Bootstrap_required )
 
   let load_full_frontier t ~root_ledger ~consensus_local_state ~max_length
-      ~ignore_consensus_local_state ~constraint_constants ~genesis_constants =
+      ~ignore_consensus_local_state ~constraint_constants
```

Comment:
remove `constraint_constants` since we passing `precomputed_values` that has constraint constants as well